portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,98 @@
+body {
+  background: #dfe0e1;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+img {
+  max-height: 60vh;
+  width: auto; 
+  border-radius: 15%;
+  opacity: .9;
+  transition-timing-function: ease-in-out;
+  transition: 250ms;
+  padding: 1vh;
+}
+
+.logo img {
+    max-height: 6vh;
+    width: auto;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 67.5vw;
+  padding: 3vh;
+}
+
+
+h1 {
+  display: block;
+  font-size: 10vh;
+  margin-block-start: 0.67em;
+  margin-block-end: 0.67em;
+  margin-inline-start: 10vw;
+  margin-inline-end: 0px;
+  font-weight: bold;
+}
+
+h1 > div {
+  font-size: 8vh; 
+  color: navy;
+}
+
+p {
+  font-size: 5vh;
 }
+#container {
+  text-align: justify;
+  justify-content: space-evenly;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+#container > div {
+  display: inline-block;
+  vertical-align: top;  
+}
+
 
-#greeting-container {
-  margin-top: 20px;
+.centered-over-img {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 6.5vh;
 }
+
+.containerBox {
+  position: relative;
+  display: inline-block;
+  vertical-align: top;
+}
+
+.text-box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  user-select: none;
+}
+
+.text-box:before { 
+  display: inline-block;
+  height: 100%;
+  vertical-align: middle;
+}
+
+container-box-caption {
+  display: inline-block;
+  font-size: 6.5vh;
+  color: white;
+  font-weight: bold;
+}
+
+img:hover { 
+  opacity: 1;
```

Comment:
This works, but I had to go look up how!  Apparently 1 is a special value for the opacity attribute that changes its z-index, thus making the TRAVEL text appear.

A simpler and less mysterious way to do this would be to simply wrap both of these in a div and write JavaScript to attach mouseenter/mouseleave events to set visibility and opacity styles on the child elements (or simply add/remove classes that have those styles).

That would also get rid of your flickering issue -- if you actually hover over the text, then the mouse is over the text, not the img, so the img loses its hover attribute... making the text disappear... thus putting the img back under the mouse... and now it is hovered again so the text reappears, creating the flickering effect on the page currently.

Reply:
Fixed the flicker phenomenon with 'pointer-events: none' instead of  'user-select: none' in the .text-box class, though I agree the implementation can be improved. I'm planning on coming back to a lot of these effects and redoing in JS once I finish these changes + content.